.rubocop.yml

```diff
@@ -0,0 +1,84 @@
+AllCops:
+  Include:
+    - '**/Rakefile'
+  Exclude:
+    - 'bin/stubs/*'
+    - 'cookbooks/**/*'
+    - 'db/schema.rb'
+    - 'vendor/**/*'
+  TargetRubyVersion: 2.1
+
+LineLength:
+  Enabled: false
+
+HashSyntax:
+  Enabled: false
+
+BracesAroundHashParameters:
+  Enabled: false
+
+BlockNesting:
+  Enabled: false
+
+AssignmentInCondition:
+  Enabled: false
```

Comment:
As discussed IRL, it should be on, and is by default so removed this in 2795027.